src/Components/Facility/DoctorCapacityForm.tsx

```diff
@@ -211,7 +211,9 @@ export const DoctorCapacityForm = (props: DoctorCapacityProps) => {
           Notification.Success({
             msg: "Doctor count added successfully",
           });
-          if (isLastOptionType) {
+          if (btnType === "Save and Exit") {
+            navigate(`/facility/${facilityId}`);
+          } else if (isLastOptionType) {
             navigate(`/facility/${facilityId}`);
```

Comment:
both the conditions are doing the same right i.e navigating to `/facility/${facilityId}`?

Reply:
Refactored the if-else statement